src/status_im/contexts/wallet/events.cljs

```diff
@@ -179,6 +179,12 @@
                                     (first derived-address-details)]))]
      {:fx [[:dispatch [:wallet/create-derived-addresses account-details on-success]]]})))
 
+(rf/reg-event-fx :wallet/select-bridge-to
+ (fn [{:keys [db]} [{:keys [token stack-id]}]]
+   {:db (-> db
+            (assoc-in [:wallet :ui :send :token] token))
```

Comment:
@briansztamfater it's right? 🤔

Reply:
regarding to the code style, it's better not having `->` of only one step